src/components/header.js

```diff
@@ -117,7 +117,7 @@ export default function Header(props) {
   const handleDrawerClose = () => {
     setOpenDrawer(false);
   };
-  const links = ["/", "/createHack", "/hacks"];
+  const links = ["/", "/create", "/hacks"];
```

Comment:
If we store this as a pairs it would be more customisable and not treat "Home" as a special case

```
const links = [
  {
    text: "Home"
    Icon: HomeIcon,
    route: "/"
  },
  {
    text: "Create Hack"
    Icon: AddIcon,
    route: "/create"
  },
  {
    text: "Hacks"
    Icon: ViewListIcon,
    route: "/hacks"
  }
]
```

ES2015 does actually also guarantee a key enumeration order (https://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order/38218582#38218582) so you could do something like

```
const links = {
  "/": "Homes",
  "/create": "Create Hack",
  "/hacks" : "Hacks"
}

const linkIcons = {
  "/": HomeIcon,
  "/create": AddIcon,
  "/hacks" : ViewListIcon
}
```

And use `Object.entries` to go through them, and access the icons through `linkIcons[key]`

although I think i prefer the first option as it keeps it all together and is a bit more flexible if you wanted the same route twice with different names (for some reason).